py3status/modules/hueshift.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+"""
+Shift color temperature on the screen.
+
+Configuration parameters:
+    button_down: mouse button to change color temperature (default 5)
+    button_toggle: mouse button to toggle color temperature (default 1)
+    button_up: mouse button to change color temperature (default 4)
+    command: specify blueshift, redshift, or sct to use, otherwise automatic
+        (default None)
+    delta: specify interval value to change color temperature (default 250)
+    format: display format for this module
+        *(default '\?if=disabled&color=bad \u263c!'
+        '|\?color=hue \u263c {temperature}K')*
+    maximum: specify maximum color temperature to use (default 25000)
+    minimum: specify minimum color temperature to use (default 1000)
+    thresholds: specify color thresholds to use
+        (default [(6499, '#f6c'), (6500, '#ff6'), (6501, '#6cf')])
+
+Control placeholder:
+    {disabled} a boolean based on pgrep processing data
+
+Format placeholder:
+    {temperature} color temperature, eg 6500
+    {command} command name, eg blueshift, redshift, sct
+
+Color options:
+    color_bad: hueshift disabled due to running processes
+
+Color thresholds:
+    hue: print a color based on the value of color temperature
+
+Requires:
+    blueshift: an extensible and highly configurable alternative
+    redshift: program to adjust the color temperature of your screen
+    sct: set color temperature
+
+Suggestions:
+    campfire: 4500
+    dust storm on mars: 2000
+    coffee free all nighter: 8000
+
+Examples:
+```
+# good rules; show command name; don't colorize everything
+hueshift {
+    format = '[\?if=disabled&color=bad \u263c!|[\?color=hue&show \u263c '
+    format += '{command}] {temperature}K]'
+    minimum = 3000
+    maximum = 10000
+}
+```
+
+@author lasers
+
+SAMPLE OUTPUT
+# warm
+{'full_text': u'\u263c 3000K', 'color': #f3c},
+
+# neutral
+{'full_text': u'\u263c 6500K', 'color': #ff3}
+
+# cool
+{'full_text': u'\u263c 10000K', 'color': #3cf},
```

Comment:
these are all wrong! you need to quote the colour values - this is a json string.  I also think colours should always be like `#XXXXXX` but that's fine when I finish cleaning up my new docstring stuff (a) I'll change them all automatically and (b) I'll add tests to make them illegal ;)

Anyhow for now just make them valid JSON